dd4hep.spec

```diff
@@ -1,20 +1,8 @@
-### RPM external dd4hep v01-19x
+### RPM external dd4hep %{dd4hep_version}
 
-%define tag cc335b34e9eb2825ab18e20c531be813a92d141f
-%define branch master
-%define github_user AIDASoft
-%define keep_archives true
-
-Source: git+https://github.com/%{github_user}/DD4hep.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}.tgz
-BuildRequires: cmake
-Requires: root boost clhep xerces-c geant4
-
-%prep
-
-%setup -n %{n}-%{realversion}
+## IMPORT dd4hep-common
```

Comment:
`## IMPORT dd4hep-common` should come before `### RPM external dd4hep %{dd4hep_version}` line otherwise rpmbuild will complain about missing `%{dd4hep_version}` definition